js/views/settingsVw.js

```diff
@@ -236,6 +239,22 @@ module.exports = pageVw.extend({
         placeholder_text_multiple: window.polyglot.t('chosenJS.placeHolderTextMultiple')
       });
 
+      var connectedPeers = ""
+      $.ajax({
+        url: self.serverUrl+ "routing_table",
+        success: function(data){
+          self.$el.find('span.js-numConnectedPeers').text(data.length);
+          data.forEach(function (peer) {
+            connectedPeers += (peer['ip'] + ":" + peer['port'] + "\n");
+          })
+          self.$el.find('span.js-connectedPeers').text(connectedPeers);
+          $('.js-connectedPeers').hide();
+        },
+        error: function(){
+          self.$el.find('.js-numConnectedPeers').text("Call to peers API failed.");
```

Comment:
Can we please make this translatable text? TO do that, you would add an entry into the en-US.json file and then you can ouput the translation via `window.polyglot.t(<translation-string>)`.